CONTRIBUTING.md

```diff
@@ -239,6 +241,18 @@ Before submitting a pull request, please make sure that:
 3. Comments are rendered correctly in the `task godocs` output.
 4. `task all` passes.
 
+#### Submitting PR
+
+1. If the pull request is related to some issues, please mention the issue number in the pull request **description** like `Closes #{issue_number}.` or `Closes org/repo#{issue_number}.`(You can just follow the pull request template).
+   If you propose a tiny fix, there is no needed to create a new issue.
+2. In the pull request review conversations, please either leave a new comment or resolve (close) the conversation, which ensures the other people can read all comments.
+   But **do not** do that simultaneously.
+3. During your development, commit messages (both titles and bodies) are not important and can be "WIP" or anything else.
+   All commits are always squashed on merge by GitHub, so there is **no need** to squash them manually, amend them, and/or do force pushes.
+   But notice that the autogenerated GitHub's squash commit's body **should be** manually replaced by "Closes #{issue_number}.".
+
+For more details about long-term contributions, please read [PROCESS.md](.github/PROCESS.md)
```

Comment:
It is more about long-term contributors (people), not contributions. I think we should say that.